app/controllers/devise_token_auth/sessions_controller.rb

```diff
@@ -55,6 +55,17 @@ def destroy
         user.tokens.delete(client)
         user.save!
 
+        if DeviseTokenAuth.cookie_enabled
+          cookie_domain = get_cookie_domain
+          if cookie_domain.present?
+            # If a cookie is set with a domain specified then it must be deleted with that domain specified
+            # See https://stackoverflow.com/a/6244724/1747491
+            cookies.delete(DeviseTokenAuth.cookie_name, domain: cookie_domain)
```

Comment:
This line is unreached by specs but it could be ok

Reply:
This conditional will be moot now, see [this recent commit](https://github.com/lynndylanhurley/devise_token_auth/pull/1453/commits/536340b3e1861fef0e595158b3588b747d765898).